postgres-appliance/scripts/configure_spilo.py

```diff
@@ -256,7 +256,7 @@ def deep_update(a, b):
     {{/SSL_CRL_FILE}}
     ssl_cert_file: {{SSL_CERTIFICATE_FILE}}
     ssl_key_file: {{SSL_PRIVATE_KEY_FILE}}
-    shared_preload_libraries: 'bg_mon,pg_stat_statements,pgextwlist,pg_auth_mon,set_user'
+    shared_preload_libraries: 'bg_mon,pg_stat_statements,pgextwlist,pg_auth_mon,set_user,pg_mon'
```

Comment:
It should be done conditionally, i.e. [here](https://github.com/zalando/spilo/pull/456/files#diff-5e29afdf8025daafbc08cc254eb83f9fR35-R40) and [here](https://github.com/zalando/spilo/blob/feature/pg13/postgres-appliance/bootstrap/pg_upgrade.py#L20-L25). Preferably only for version 13.